mobile/lib/repositories/asset_media.repository.dart

```diff
@@ -43,4 +43,15 @@ class AssetMediaRepository implements IAssetMediaRepository {
     asset.local = local;
     return asset;
   }
+
+  @override
+  Future<String?> getOriginalFilename(String id) async {
+    final entity = await AssetEntity.fromId(id);
+
+    if (entity == null) {
+      return null;
+    }
+
+    return await entity.titleAsync;
```

Comment:
Might want a comment here explaining why it specifically needs to be titleAsync (vs eg filename)